### DISCUSSION TITLE
Renaming Pocketbase CLI

### DISCUSSION THREAD

**Discussion by vipas84** [pocketbase/pocketbase#4803] [Q&A] [answered]
When using Pocketbase as a framework I need to print out custom messages and commands, not in addition but instead of default ones? Is there any way to do so?

**Comment by ganigeorgiev** [comment]
I don't understand what do you mean by _"print out custom messages and commands"_.

Please check:

- Go:   https://pocketbase.io/docs/go-console-commands/
- JSVM: https://pocketbase.io/docs/js-console-commands/

If that doesn't help, please elaborate more clearly what you are trying todo to understand the bigger picture of your use case.

**Reply by vipas84** [reply]
Hello Gani, thank you for your prompt reply!
Sorry for poor description, let me try one more time
My use case:
- I use pocketbase as a framework and compile binary with the "name of my solution".
- when I use CLI, it still mentions "Pocketbase CLI"

Simplified question: is it possible to rename "Pocketbase CLI" to "[my solution name] CLI"?

**Reply by ganigeorgiev** [reply]
You can modify the cli description by changing `app.RootCmd.Short`, eg:

```go
app := pocketbase.New()

app.RootCmd.Short = "your app description"
// if you also want to change the name from the usage examples (default to os.Args[0]):
// app.RootCmd.Use = "yourapp"

...
```

For all fields, you can refer to the [`cobra.Command` docs](https://pkg.go.dev/github.com/spf13/cobra#Command).

**Reply by vipas84** [reply, marked as answer]
thanks a lot!
